### ISSUE TITLE
Android TV

### ISSUE THREAD

**Issue by denilsonsa** [IvanVolosyuk/diskusage#62]
[DiskUsage](https://play.google.com/store/apps/details?id=com.google.android.diskusage) is listed at the Play Store, but is not available for Android TVs. However, if you manually sideload and install the app (let's say, [from F-Droid](https://f-droid.org/en/packages/com.google.android.diskusage/)), the app works fine on Android TV and can be controlled even without a touch screen (i.e. can be controlled with the arrow keys from a remote control).

Thus, it would be nice if you could add support for Android TV into your app.

I'm not sure what's needed (probably some change in the manifest with `android.software.leanback` and `android.intent.category.LEANBACK_LAUNCHER`, plus some launcher image, and maybe some changes for the guidelines of the Google Play store).

**Comment by 89z** [comment]
No, you cant do that. As soon as you add `android.software.leanback`, it breaks the app for phones. So two apps would be needed, similar to these:

- https://play.google.com/store/apps/details?id=com.google.android.youtube
- https://play.google.com/store/apps/details?id=com.google.android.youtube.tv

**Comment by draxx31** [comment]
Hello, 

I tried today to use DiskUsage on a Xiaomi mi box S 2ng gen (android TV 11). 

The application starts well and works but I cannot navigate deeply because up and down action from my remote change the view.

It will be very interesting to have this app ported on Android TV because there is no such tool available on it.
